### PR TITLE
[Website] Removed posts, rewrote product eng ones

### PR DESCRIPTION
**Pull request by bkase**
Hack for making styles acceptable in this quick deploy

Already deployed as we're about to send out the newsletter.

Screen Shot 2019-06-20 at 5 12 13 AM

### CODE REVIEW

File: frontend/website/jobs/frontend-engineer.markdown [hunk 1]
```diff
@@ -1,32 +1,51 @@
...
 
-We’re working on technologies with the potential to reimagine social structures. We believe it’s important to incorporate diverse perspectives from conception through realization.
+We’re seeking a frontend product engineer who is also excited by building beautiful products. While it's hard to say exactly what we'll work on given that we are such an early-stage startup, we expect projects to include:
+
+- Wallet software that is a joy to use, with innovative onboarding and account recovery mechanisms.
+- The product side of developer tools and SDKs
+- Imaginative cryptocurrency applications that showcase what’s possible with a lightweight blockchain
+
+
+# Requirements
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: nit: header feels too big relative to job title - maybe shift down one header size

File: frontend/website/jobs/frontend-engineer.markdown [hunk 2]
```diff
@@ -1,32 +1,51 @@
 ---
-title: Senior Frontend Engineer
-date: 2019-02-15
+title: Product Engineer (Frontend)
+date: 2019-06-20
 ---
-O(1) Labs is aiming to develop the first cryptocurrency protocol that can deliver on the promise of supporting real-world applications and widespread use. Our team is based in San Francisco, and we are funded by top investors (including Polychain, Metastable, Max Levchin, and Naval Ravikant).
 
-We're bringing Coda Protocol to market, a cryptocurrency that compresses the blockchain from hundreds of gigabytes down to the size of a few tweets. It can scale to thousands of transactions per second and millions of users while remaining decentralized enough for cellphones to be fully verifying nodes.
+Help us build accessible, usable, delightful products on top of the Coda Cryptocurrency Protocol!
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: nit: remove cryptocurrency

File: frontend/website/jobs/frontend-engineer.markdown [hunk 3]
```diff
@@ -1,32 +1,51 @@
...
 
-We’re working on technologies with the potential to reimagine social structures. We believe it’s important to incorporate diverse perspectives from conception through realization.
+We’re seeking a frontend product engineer who is also excited by building beautiful products. While it's hard to say exactly what we'll work on given that we are such an early-stage startup, we expect projects to include:
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: nit can remove frontend here - should be implicit based on the job title and the job reqs.  Also remove "such" from next line

File: frontend/website/jobs/frontend-engineer.markdown [hunk 4]
```diff
@@ -1,32 +1,51 @@
...
+
+
+# Requirements
+
+
+- An internship or 1+ years engineering experience
+    - Bonus: Worked on a product team doing frontend work
+- Willingness to work on-site in a San Francisco based office
+- Exposure to a declarative frontend framework, such as React, Angular, Vue, Elm etc.
+    - Bonus for experience with React Hooks or GraphQL
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: mirror previous bonus syntax

Bonus: Experience with React Hooks or GraphQL

File: frontend/website/jobs/frontend-engineer.markdown [hunk 5]
```diff
@@ -1,32 +1,51 @@
...
 
-This is a chance to join a small, collaborative team and have a ton of independence while working on fascinating cross-disciplinary problems that span cryptography, engineering, product design, economics, and sociology. We also offer competitive compensation both in salary and equity as well as top-of-the-market benefits.
 
-We're seeking a senior frontend engineer who is also excited by building beautiful products. There is no need to have prior experience in crypto, and we’re hiring engineers to work on a bunch of exciting projects including:
+# Grab Bag (at least one of the following)
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: mirror with senior role

"Nice to have"

File: frontend/website/jobs/frontend-engineer.markdown [hunk 6]
```diff
@@ -1,32 +1,51 @@
...
-* Imaginative cryptocurrency applications that showcase what’s possible with a lightweight blockchain
 
-Requirements:
+- Familiarity with functional programming, know your way around map and reduce.
+- CSS chops, should be comfortable with Flexbox.
+- Experience working on multiple collaborative projects with a product team containing at least one additional developer or designer (or larger team-based projects at university or a bootcamp program).
+- Exposure to working with cryptocurrencies
+- Experience with ReasonML, OCaml, Elm, or Purescript
+- Early stage startup experience
+- Experience with contributing to Open Source
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: open source need not be caps

File: frontend/website/jobs/frontend-engineer.markdown [hunk 7]
```diff
@@ -1,32 +1,51 @@
...
 
-* Frontend and product development experience
 
-There are no hard requirements, but any one of the following are excellent qualifications:
+# About
```
Comment (o1pranay) [inline frontend/website/jobs/frontend-engineer.markdown]: About us

File: frontend/website/src/Render.re [hunk 8]
```diff
@@ -107,14 +107,13 @@ module Router = {
...
 
 let jobOpenings = [|
-  ("engineering-manager", "Engineering Manager (San Francisco)."),
   ("product-manager", "Product Manager (San Francisco)."),
-  ("senior-frontend-engineer", "Senior Frontend Engineer (San Francisco)."),
-  ("protocol-engineer", "Senior Protocol Engineer (San Francisco)."),
   (
-    "director-of-business-development",
-    "Director of Business Development (San Francisco).",
+    "senior-frontend-engineer",
```
Comment (o1pranay) [inline frontend/website/src/Render.re]: maybe this can also say senior-product-engineer

Comment (bkase) [reply frontend/website/src/Render.re]: I think we need to keep this `senior-frontend-engineer.html` so we don't have dead links elsewhere (I guess the right solution would be to have both or a redirect, but that would be more eng effort)

**Pull request merged**